core/finance/metrics/tests/test_approved_funds.py

```diff
@@ -0,0 +1,36 @@
+import unittest
```

Comment:
Test files are now put inside folder `/core/tests`, please, move this file to that path.

Reply:
Done.